src/learning_service/prometheus.yml

```diff
@@ -0,0 +1,12 @@
+global:
```

Comment:
There is no need for any kind of Prometheus configuration in the repo - application is only exposing the metrics, actual scraping is done on the infrastructure side.